histomicsui/web_client/panels/DrawWidget.js

```diff
@@ -114,11 +114,16 @@ var DrawWidget = Panel.extend({
                 name,
                 opts: this._editOptions,
                 drawingType: this._drawingType,
-                collapsed: this.$('.s-panel-content.collapse').length && !this.$('.s-panel-content').hasClass('in')
+                collapsed: this.$('.s-panel-content.collapse').length && !this.$('.s-panel-content').hasClass('in'),
+                firstRender: true
             }));
             this.$('.h-dropdown-content').collapse({toggle: false});
         }
-        this.$('button.h-draw[data-type]').removeClass('active');
```

Comment:
Is there a reason you removed this line?

Reply:
No! Thank you for pointing that out!

See 0e990b1